zerver/migrations/0501_merge_20240228_1140.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 4.2.10 on 2024-02-28 11:40
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0497_archivedmessage_has_reaction_message_has_reaction"),
+        ("zerver", "0500_realm_zulip_update_announcements_stream"),
+    ]
+
+    operations = []
```

Comment:
You should be renumbering migrations when you rebase, not creating these merge migrations as Django's tooling suggests. See https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html for important documentation on this topic.